models/reports_struct.go

```diff
@@ -0,0 +1,19 @@
+package models
+
+type ReportListing struct {
+	Kind string            `json:"kind"`
+	Data ReportListingData `json:"data"`
+}
+
+type ReportListingData struct {
+	Modhash  string               `json:"modhash"`
+	Dist     float64              `json:"dist"`
+	Children []ReportListingChild `json:"children"`
+}
+
+type ReportListingChild struct {
+	Kind   string      `json:"kind"`
+	Data   interface{} `json:"data"`
```

Comment:
Do we just not care about this field? (totally ok if we don't)

Reply:
I apologise if it is obvious, but I am not sure which field you are referring to.
If it is Kind, then we do care about it, because it is used to identify whether the Report/Modqueue entry is a thread or a comment.
If it is Data, then we do care about it, I opted to choose `interface{}` as the data type as it will be later cast to a `CommentListingDataChildrenData` if it is a comment and `PostListingChildData` if it is a thread.

Reply:
I see! When I add a comment to a line, I guess Github doesn't really highlight it. It's usually the very last line in the code review window, so I was referring to `Data`. Good stuff, I see where the casting is done, it's good!